hotel-frontend/src/assets/product-manage.css

```
@@ -0,0 +1,780 @@
+.product-manage-page .page-size-area {
+  display: flex;
+  align-items: center;
+  gap: 7px;
+  color: #756a60;
+  font-size: 13px;
+}
+
+.product-manage-page .page-size-select {
+  padding: 6px 10px;
+
+  border: 1px solid #d8d0c5;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)